jib-gradle-plugin/README.md

```diff
@@ -162,7 +162,7 @@ Field | Type | Default | Description
 `to` | [`to`](#to-closure) | *Required* | Configures the target image to build your application to.
 `container` | [`container`](#container-closure) | See [`container`](#container-closure) | Configures the container that is run from your built image.
 `useProjectOnlyCache` | `boolean` | `false` | If set to `true`, Jib does not share a cache between different Maven projects.
-`allowInsecureRegistries` | `boolean` | `false` | If set to true, Jib uses HTTP as a fallback for registries that do not support HTTPS or whose certificates cannot be verified. Leaving this parameter set to `false` is strongly recommended, since communication with insecure registries is unencrypted and visible to others on the network.
+`allowInsecureRegistries` | `boolean` | `false` | If set to true, Jib can ignore HTTPS certificate errors, or fall back to HTTP as a last report. Leaving this parameter set to `false` is strongly recommended, since HTTP communication is unencrypted and visible to others on the network, and insecure HTTPS is no better than plain HTTP.
```

Comment:
s/can ignore/ignores/
s/report/resort/

Isn't it "and falls back"?  Worth adding: _If accessing a registry with a self-signed certificate, add the certificate to your Java runtime's trusted keys._ and link to the self-signed document (`docs/self_sign_cert.md`).

Reply:
"ignores certificate errors and falls back to HTTP" reads to me like it will fall back to HTTP, so I think I'll go with "or may fall back to HTTP as a last resort".

About adding the certificate, how about

"If accessing a registry with a self-signed certificate, adding the certificate to your Java runtime's trusted keys may be another option."

? Personally, I don't find disguise an unverifiable, temporary certificate as a trusted one and forcing that at the system level always nice. It still seems like an interim solution.